pkgs/servers/monitoring/grafana/plugins/grafana-googlesheets-datasource/default.nix

```diff
@@ -0,0 +1,13 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "grafana-googlesheets-datasource";
+  version = "1.2.5";
+  zipHash = "sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=";
```

Comment:
This zipHash repeats a few times and I can't figure out how to get the update script to fetch the right thing. It looks like `nix-prefetch-url` fails with a HTTP 400 for these, while it works fine if I just curl.